druid/src/app.rs

```diff
@@ -38,11 +36,22 @@ pub struct AppLauncher<T> {
     ext_event_host: ExtEventHost,
 }
 
+/// Defines how a windows size should be determined
+#[derive(Copy, Clone, Debug)]
+pub enum WindowSizePolicy {
+    /// Use the content of the window to determine the size
```

Comment:
```suggestion
    /// Use the content of the window to determine the size.
    ///
    /// If you use this option, your root widget will be passed infinite constraints;
    /// you are responsible for ensuring that your content picks an appropriate size.
```

This is my understanding, not sure if it's correct? But seems worth writing down.